cloudflare/resource_cloudflare_custom_ssl.go

```diff
@@ -224,17 +234,24 @@ func resourceCloudflareCustomSslDelete(d *schema.ResourceData, meta interface{})
 }
 
 func resourceCloudflareCustomSslImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	client := meta.(*cloudflare.API)
+
 	// split the id so we can lookup
 	idAttr := strings.SplitN(d.Id(), "/", 2)
-
+	var zoneName string
```

Comment:
Let's keep it at zoneID as we're in the process of deprecating zoneName from this provider due to the non-unique nature of it in Cloudflare.

Reply:
cool, updated